tools/monitor/colors.py

```diff
@@ -0,0 +1,70 @@
+import curses
+import os
+import json
+
+class Colors:
+    def __init__(self):
+        self._config = self._try_load_configuration()
+        self._define_default_colors()
+
+        self.DCYAN = self._get_color('DCYAN', 1)
+        self.DRED = self._get_color('DRED', 2)
+        self.DBLUE = self._get_color('DBLUE', 3)
+        self.RED = self._get_color('RED', 4)
+        self.GREEN = self._get_color('GREEN', 5)
+        self.YELLOW = self._get_color('YELLOW', 6)
+        self.DYELLOW = self._get_color('DYELLOW', 7)
+        self.CYAN = self._get_color('CYAN', 8)
+
+    def _try_load_configuration(self):
+        config_file_name = 'monitor_colors.json'
+        path = os.path.join(os.getcwd(), config_file_name)
+        if not os.path.isfile(path):
+            return None
+
+        with open(path, 'r') as f:
+            data = json.load(f)
+            return data
+
+    def _define_default_colors(self):
+        curses.init_pair(1,curses.COLOR_CYAN,0)
+        curses.init_pair(2,curses.COLOR_RED,0)
+
+        if curses.COLORS > 8:
+            curses.init_color(20, 0 ,0 ,850)
+            curses.init_pair(3,20 , -1)
```

Comment:
34 i 35 psują pod win7 :) jezeli łatwo da się poprawić, to można - jak nie, to dopisać że nie działa na win7 i też dla mnie git

Reply:
poprawione